.golangci.yml

```diff
@@ -33,5 +33,3 @@ linters-settings:
     disable:
       - fieldalignment
       - shadow
-  makezero:
```

Comment:
Any reason we're removing this?

Reply:
It's the default value for this linters option. No need to explicitly define it. 👍